internal/proto/controller/api/services/v1/credential_library_service.proto

```diff
@@ -83,43 +86,43 @@ service CredentialLibraryService {
 }
 
 message GetCredentialLibraryRequest {
-  string id = 1;
+  string id = 1; // @gotags: `class:"public"`
 }
 
 message GetCredentialLibraryResponse {
-  resources.credentiallibraries.v1.CredentialLibrary item = 1;
+  resources.credentiallibraries.v1.CredentialLibrary item = 1; // @gotags: `class:"public"`
```

Comment:
I believe this doesn't need to be tagged since it's a struct.

Reply:
Ah, yep. I forgot about this file when I fixed up the other one.